src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order(new ArrayList<>(shoppingCart.getTickets()),
+                LocalDateTime.now(), shoppingCart.getUser());
+        shoppingCartService.clearShoppingCart(shoppingCart);
+        return orderDao.add(order);
```

Comment:
let's create an order first, and then clear the cart because we can clear it and then face some error in the process of creating an order, our service method is not transactional yet so we will lose data